src/MessagePack/SequenceReaderExtensions2.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Buffers
+{
+    internal static partial class SequenceReaderExtensions
+    {
+        /// <summary>
+        /// Reads an <see cref="sbyte"/> from the next position in the sequence.
+        /// </summary>
+        /// <param name="reader">The reader to read from.</param>
+        /// <param name="value">Receives the value read.</param>
+        /// <returns><c>true</c> if there was another byte in the sequence; <c>false</c> otherwise.</returns>
+        public static bool TryRead(ref this SequenceReader<byte> reader, out sbyte value)
+        {
+            if (reader.TryRead(out byte byteValue))
```

Comment:
These methods were changed to call the `TryRead` extension methods rather than the `TryRead<T>` which is unavailable in netcoreapp3.0

Reply:
Probably not an issue anyway though, since we're keeping our `SequenceReader<T>` type.